.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,22 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    steps:
+    - uses: actions/download-artifact@v2
+    - uses: actions/checkout@v2
+  npm:
+    runs-on: ubuntu-latest
+    container:
+      image: node:lts
+    steps:
```

Comment:
The one thing I'm thinking might be missing here is the ssh keys... @gbockus-sf can you think of reason we would need those here for the publish to work?

Reply:
Reaching back to remember how this works.  We need the ssh keys when we need to merge into protected branch like main.  If for this release process we don't have to do any commits we probably don't need the ssh keys.